src/main/java/coydir/model/person/Leave.java

```diff
@@ -0,0 +1,61 @@
+package coydir.model.person;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Represents a Leave in the database.
+ * Guarantees: immutable; name is valid as declared in {@link #isValidLeave}
+ */
+public class Leave {
+    private static final DateTimeFormatter FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd");
```

Comment:
We use this instead of "dd-MM-yyyy"? I would think "dd-MM-yyyy" is more intuitive/familiar.

Reply:
Hmm what do yall think? @chao890 @Kok-je @ngshijun

Reply:
I do believe that "dd-MM-yyyy" will be more intuitive. @prit3010